scripts/tools/DeliveryExecutionReport.php

```diff
@@ -0,0 +1,213 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+namespace oat\taoSystemStatus\scripts\tools;
+
+use common_exception_Error;
+use common_exception_NotFound;
+use InvalidArgumentException;
+use oat\oatbox\extension\AbstractAction;
+use oat\oatbox\log\LoggerAwareTrait;
+use oat\tao\model\taskQueue\TaskLog;
+use oat\tao\model\taskQueue\TaskLog\Broker\TaskLogBrokerInterface;
+use oat\tao\model\taskQueue\TaskLog\TaskLogFilter;
+use oat\taoAct\model\transmissionLog\TransmissionLog;
+use oat\taoAct\model\transmissionLog\TransmissionLogService;
+use oat\taoDelivery\model\execution\ServiceProxy;
+use oat\taoProctoring\model\deliveryLog\DeliveryLog;
+use oat\taoProctoring\model\monitorCache\DeliveryMonitoringService;
+use common_report_Report as Report;
+use common_ext_ExtensionsManager;
+
+/**
+ * Class DeliveryExecutionReport
+ *
+ * @package oat\taoSystemStatus\scripts\tools
+ * @author Andrei Niahrou, <Andrei.Niahrou@1pt.com>
+ */
+class DeliveryExecutionReport extends AbstractAction
+{
+    use LoggerAwareTrait;
+
+    /** @var string */
+    private $executionDeliveryId;
+
+    /**
+     * @inheritdoc
+     */
+    public function __invoke($params = [])
+    {
+        $this->init($params);
+        return $this->execute($this->executionDeliveryId);
+    }
+
+    /**
+     * @param array $params
+     * @throws
+     */
+    private function init($params = [])
+    {
+        if (!isset($params[0])) {
+            throw new InvalidArgumentException('Missing parameter');
+        }
+        $this->executionDeliveryId = $params[0];
+    }
+
+    /**
+     * Show comprehensive information about delivery execution
+     *
+     * Run example:
+     * ```
+     * sudo php index.php 'oat\taoSystemStatus\scripts\tools\DeliveryExecutionReport' http://act-pr.docker.localhost/tao.rdf#i5e28154860c9a17943be5add4f7b1b5
+     * ```
+     *
+     * @param string $executionId
+     * @return Report
+     * @throws common_exception_Error
+     * @throws common_exception_NotFound
+     */
+    private function execute($executionId)
+    {
+        $report = new Report(Report::TYPE_INFO, sprintf('Information about delivery execution %s', $executionId));
+        $extensionManager = $this->getExtensionManager();
+
+        $deliveryExecutionService = $this->getDeliveryExecutionService();
+        $deliveryExecution = $deliveryExecutionService->getDeliveryExecution($executionId);
+        $this->addSubReport($report, 'Delivery execution report', $deliveryExecution);
```

Comment:
Please create an array from `$deliveryExecution` with the following fields:
delivery_id, user_id, status, finished_at, started_at, label
And then set this array to the report's data instead of `$deliveryExecution` itself